### DISCUSSION TITLE
Minus Icon

### DISCUSSION THREAD

**Discussion by mmichealjroberts** [tailwindlabs/heroicons#376] [Ideas]
I sound like I am moaning something rotten - but there doesn't look to be a minus sign, but a plus sign. Well worth adding.

Switching from font-awesome to heroicons for applications.

**Comment by cossssmin** [comment]
Yes please! 😊 

Was just building a mobile dropdown with a +/- icon toggle with Tailwind UI, but there's no **&ndash;** icon so I have to fallback to chevrons for now :(

**Comment by Braunson** [comment]
Also requesting, while working on an FAQ design, I noticed heroicons has a + plus icon but not a matching/complimentary minus/negative icon (there is a rounded but it does not match the non-rounded plus).

**Edit** I've created my own, anyone is welcome to use it for the time being.

```
<svg class="h-6 w-6" fill="currentColor" viewBox="0 0 20 20">
	<path d="M0 9h15v2h-20z"/>
</svg>
```
